src/Symfony/Component/Validator/Constraints/IbanValidator.php

```diff
@@ -163,9 +163,6 @@ class IbanValidator extends ConstraintValidator
         'YT' => 'FR\d{2}\d{5}\d{5}[\dA-Z]{11}\d{2}', // France
     ];
 
-    /**
-     * {@inheritdoc}
-     */
```

Comment:
This is removed by fabbot.io, maybe unrelated?

Reply:
Let's keep it

Reply:
Done https://github.com/symfony/symfony/pull/54924/commits/c2223b115f4b2073e651f5fbd89d5a396203763a